packages/atomic/package.json

```diff
@@ -22,7 +22,7 @@
     "test": "stencil test --spec --e2e",
     "test:watch": "stencil test --spec --e2e --watchAll",
     "generate": "stencil generate",
-    "cypress:test": "npx cypress run",
+    "cypress:test": "npx cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e",
```

Comment:
What's that key ? Is it okay to commit that here ?

And how do people run tests locally with cypress. 
Don't they use that command ? `npm run cypress:test` ?

If so that would be recorded in the dashboards ?

Reply:
the key is to record and link with dashboard. Idea yes, we shouldn't put it here, but I plan to replace it when I start my gitaction story
when you run from local machine `npm run cypress:test` will record it in the dashboard
but if you test on local, more practical, npm run cypress:open, it will open a 'local view', and you can run only your selected test that you are working on